platform-service/src/unstract/platform_service/helper.py

```diff
@@ -26,11 +29,19 @@ def get_adapter_instance_from_db(
         Returns:
             _type_: _description_
         """
-        query = (
-            f"SELECT id, adapter_id, adapter_metadata_b FROM "
-            f'"{organization_id}".adapter_adapterinstance x '
-            f"WHERE id='{adapter_instance_id}'"
-        )
+        if check_feature_flag_status(FeatureFlag.MULTI_TENANCY_V2):
+            query = (
+                "SELECT id, adapter_id, adapter_metadata_b FROM "
+                f"{DBTableV2.ADAPTER_INSTANCE} x "
+                f"WHERE id='{adapter_instance_id}' and "
+                f"organization_id='{organization_uid}'"
+            )
+        else:
+            query = (
+                f"SELECT id, adapter_id, adapter_metadata_b FROM "
+                f'"{organization_id}".adapter_adapterinstance x '
+                f"WHERE id='{adapter_instance_id}'"
+            )
```

Comment:
@muhammad-ali-e is better to have SQLAlchemy instead of query? I think code will be much shorter if we follow SQLAlchemy. Later in refactoring we might need to refactor again

Reply:
@kirtimanmishrazipstack The entire code is same we have in older version. This Pr doesn't means any refactoring or  new codes.
Also I think we are not using sqlalchemy in any of our flask servers.